src/behaviors/Behavior.js

```diff
@@ -162,16 +236,18 @@ var Behavior = Base.extend('Behavior', {
     },
 
     clearColumns: function() {
-        var schema = this.dataModel.schema,
+        var schema = this.schema,
             tc = this.treeColumnIndex,
             rc = this.rowColumnIndex;
 
         schema[tc] = schema[tc] || {
+            index: tc,
             name: 'Tree',
             header: 'Tree'
```

Comment:
Per @nmichaud: Default tree column name/header should not be hard-coded; it should be settable.

Reply:
I will move the strings to `Behavior.prototype.defaultTreeColumnName` and `.defaultTreeColumnHeader`, which can be overridden by app.